e2e/func-e_use_test.go

```diff
@@ -73,10 +73,14 @@ func TestFuncEUse_UnknownMinorVersion(t *testing.T) {
 	stdout, stderr, err := funcEExec("use", v)
 
 	require.EqualError(t, err, "exit status 1")
-	require.Empty(t, stdout)
-	require.Equal(t, moreos.Sprintf("error: couldn't find the latest patch for version %s\n", v), stderr)
+	require.Regexp(t, `^looking up the latest patch for Envoy version 1.1\r?\n$`, stdout)
+	stderrPattern := fmt.Sprintf("^error: https://.*json does not contain an Envoy release for version 1.1 on platform %s/%s\n$", runtime.GOOS, runtime.GOARCH)
+	require.Regexp(t, stderrPattern, stderr)
 }
 
+// TODO: this test overuses bandwidth, making it tedious especially on slow networks.
+// Parse the envoy-versions.json to dynamically select the version before LastKnownEnvoyMinor if it isn't consistent.
```

Comment:
this is super annoying and I noticed it earlier, just it didn't really burn me as much until in a hotel. #393 discusses this more broadly and the solution would be change `make check` to ensure LastKnownEnvoy is consistent across all supported platforms (and have at least two patches, so min patch .1). That way, we don't need to redundantly download a second copy of an arbitrary version here. Since LastKnownEnvoy is advisory only (ex only in help docs and e2e tests) limiting this as I described shouldn't really have any impact.